ExeterTeam25/group-software-project
# Upgrade user

Can now request to become  a gamemaster

To test:

Login as a user that is not a gamemaster. Go to profile and click "Become a game master".
Enter a reason why.

Logout and login as a gamemaster.
There should be a new item in the sidebar with all the requests to become a game master
Accept a request and go into admin to check they are now a game master
Login as the new gamemaster and check the sidebar now shows the requests for that account as well
